tests/util/translator/generate/elements/dropdown/dropdown-schema-generator.spec.tsx

```diff
@@ -63,3 +92,9 @@ describe("DropdownSchemaGenerator", () => {
         expect(generatedSchema).toStrictEqual(expectedSchema);
     });
 });
+
+// =============================================================================
+// HELPERS
+// =============================================================================
+const elementId = "mockId123";
+const label = "dropdown-field";
```

Comment:
as these constants are outside any functions, we want to use UPPER_SNAKE_CASE 

trouble you to fix for the other tests as well

```suggestion
const ELEMENT_ID = "mockId123";
const LABEL = "dropdown-field";
```

Reply:
Fixed for generators. Will do so for parsers in corresponding branch.